src/Pos/Genesis.hs

```diff
@@ -185,7 +185,9 @@ genesisUtxo :: StakeDistribution -> Utxo
 genesisUtxo sd =
     M.fromList . zipWith zipF (stakeDistribution sd) $ genesisAddresses
   where
-    zipF coin addr = (TxIn (unsafeHash addr) 0, (TxOut addr coin, []))
+    zipF coin addr =
+        ( TxIn (unsafeHash addr) 0
+        , (TxOutAux {toaOut = TxOut addr coin, toaDistr = []}))
```

Comment:
I think it's a stylistic matter and it's subjective. We don't have precise rules about it, so I think it's fine to leave it as it is now.